GenderPayGap.WebUI/Views/Components/PaginationLinks/GpgPaginationLinksViewModel.cs

```diff
@@ -1,17 +1,17 @@
 namespace GenderPayGap.WebUI.Views.Components.PaginationLinks
 {
-    public class PaginationLinksViewModel

```

Comment:
The design library has a pagination component. Can we get rid of this and use that instead?